server/repository/src/mock/invoice.rs

```diff
@@ -116,6 +116,20 @@ pub fn mock_outbound_shipment_no_lines() -> InvoiceRow {
     })
 }
 
+pub fn mock_new_outbound_shipment_no_lines() -> InvoiceRow {
+    inline_init(|r: &mut InvoiceRow| {
+        r.id = String::from("new_outbound_shipment_no_lines");
+        r.name_id = String::from("name_store_a");
+        r.store_id = String::from("store_c");
+        r.invoice_number = 3;
+        r.r#type = InvoiceRowType::OutboundShipment;
+        r.status = InvoiceRowStatus::New;
+        r.comment = Some("Sort comment test ac".to_owned());
```

Comment:
Typically with inlint_init, we just include the required fields (comment, their_reference, created_dateime, invoice_number are. not really needed

Reply:
thanks